docs/usage.md

```diff
@@ -148,3 +148,70 @@ Connect to nix-bitcoin node through ssh Tor Hidden Service
 	```
 
 6. Now you can run `nixops deploy -d bitcoin-node` and it will connect through the ssh tunnel you established in step iv. This also allows you to do more complex ssh setups that `nixops ssh` doesn't support. An example would be authenticating with [Trezor's ssh agent](https://github.com/romanz/trezor-agent), which provides extra security.
+
+Initialize a Trezor for Bitcoin Core's Hardware Wallet Integration
+---
+
+1. Enable Trezor in `configuration.nix`
+
+	Change
+	```
+	# services.hardware-wallets.trezor = true;
+	```
+	to
+	```
+	services.hardware-wallets.trezor = true;
+	```
+
+2. Deploy new `configuration.nix`
+
+	```
+	nixops deploy -d bitcoin-node
+	```
+
+3. Check that your nix-bitcoin node recognizes the Trezor
+
+	```
+	nixops ssh operator@bitcoin-node
+	lsusb
+	```
+	Should show something relating to your Trezor
+
+3. Start the Trezor in bootloader mode
```

Comment:
Nit: it should be noted that 3. 4. and 5. only need to be done if you haven't initialized your trezor already or if you run outdated firmware.